packages/analytics-browser/src/attribution/campaign-tracker.ts

```diff
@@ -0,0 +1,88 @@
+import { createIdentifyEvent, Identify } from '@amplitude/analytics-core';
```

Comment:
dedicated module to emit campaign related data. handles
* persisting campaign data for future comparison
* determining if campaign is new
* converting campaign data to identify event
* tracking event